core/utils/silly-i18n/src/index.js

```diff
@@ -56,11 +53,21 @@ export class I18n {
         return Object.keys(this._translations);
     }
 
+    /**
+     * Returns the automatically determined (in constructor) locale string.
```

Comment:
Nit: Guess this comment needs updating now that it is not necessarily automatically determined anymore. If this property getter is needed at all now.